bot_impl.py

```diff
@@ -5620,20 +5634,26 @@ async def on_message(message):
 
                     vt = int(argument == "yes" or argument == "y")
 
+                    if voudon_in_play and voudon_in_play != person and not person.is_ghost and vt > 0:
```

Comment:
ah here you do actually care about the voudon itself.  I'd maybe rename the variable to `in_play_voudon` rather than `voudon_in_play` then.  The frist sounds like a boolean, the second sounds like a nullable player

Reply:
Thinking more, I'd probably stick to the boolean and just update this to be:

```py
if voudon_in_play and player.character.role_name != "Voudon" and not person.is_ghost and vt > 0
```